### ISSUE TITLE
Navigation links and event meta on The Event Calendar single event pages erroneously have SLB attributes

### ISSUE THREAD

**Issue by andylepp** [archetyped/simple-lightbox#833]
My site is an intranet so I can't share directly with you, but SLB is adding attributes to non-image links in event pages generated by The Events Calendar (Pro) plugin. I've temporarily hidden the links which have issues, but this may indicate a wider issue.
![image](https://user-images.githubusercontent.com/3668595/95311334-5f735a80-0885-11eb-959f-70a7c3aa2dbe.png)
![image](https://user-images.githubusercontent.com/3668595/95311389-76b24800-0885-11eb-85f8-a2aa6c2eccf5.png)

**Comment by archetyped** [comment]
Please provide a link to the specific plugin you are experiencing a conflict with so that it can be evaluated.

Thank you.

**Comment by andylepp** [comment]
https://wordpress.org/plugins/the-events-calendar/

**Comment by andylepp** [comment]
On further investigation I think I've sorted this issue out. 

The file which was popping up was named 0.jpg (I've no idea why either), so I've renamed it to a more appropriate name and the issue has stopped. 

The common link was that the URLs of where this was occurring all contained the word "event" in some form. I guess there's no conflict if this action has sorted it out.

BTW - I tried some alternative lightbox plugins and they were all terrible in comparison to SLB - thanks for all the effort!
